SharpRepository.XmlRepository/XmlRepositoryBase.cs

```diff
@@ -27,14 +27,7 @@ internal XmlRepositoryBase(string storagePath, ICachingStrategy<T, TKey> caching
         private void Initialize(string storagePath)
         {
             _items = new List<T>();
-            _storagePath = storagePath;
-
-            if (!_storagePath.EndsWith(@"\"))
-            {
-                _storagePath += @"\";
-            }
-
-            _storagePath = String.Format("{0}{1}.xml", _storagePath, TypeName);
+            _storagePath = storagePath; _storagePath = Path.Combine(storagePath, TypeName + ".xml");
```

Comment:
obviously  Path.Combine is a better solution!

but why assign _storagePath  in same lane 2 times?
` _storagePath = storagePath;` is useless

Reply:
```suggestion
            _storagePath = Path.Combine(storagePath, TypeName + ".xml");
```